src/openrct2/Game.cpp

```diff
@@ -96,8 +96,8 @@ static GAME_COMMAND_CALLBACK_POINTER * const game_command_callback_table[] = {
     nullptr,
     nullptr,
     nullptr,
-    game_command_callback_pickup_guest,
-    game_command_callback_pickup_staff
+    nullptr,
+    nullptr
```

Comment:
Now this table is empty, might as well get rid of it if it isn't too much trouble.

Reply:
I thought about doing that but then decided I'd make a second pr for that as I've a feeling it won't be as simple as just deleting this bit